stroom-search/stroom-search-elastic/src/main/java/stroom/search/elastic/ElasticClientConfig.java

```diff
@@ -0,0 +1,48 @@
+package stroom.search.elastic;
+
+import stroom.util.shared.AbstractConfig;
+import stroom.util.shared.IsStroomConfig;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import org.elasticsearch.client.RestClientBuilder;
+
+public class ElasticClientConfig extends AbstractConfig implements IsStroomConfig {
```

Comment:
Can you add `@JsonPropertyOrder(alphabetic = true)` on all config classes as this ensures a consistent serialised form.